src/main/java/seedu/duke/command/Command.java

```diff
@@ -6,18 +6,13 @@
 public abstract class Command {
 
 
-    protected Hashtable<String, String> parametersInHT = new Hashtable<>();
-    protected ArrayList<String> parametersInAL = new ArrayList<>();
+    protected Hashtable<String, String> parameters = new Hashtable<>();
```

Comment:
No need for `new Hashtable<>();` since you are overwriting it at line 15.

Reply:
Updated this issue.